### PR TITLE
Celestia.Celestia version 1.6.4

### PR DESCRIPTION
**Pull request by vedantmgoyal9**
Checklist for Pull Requests
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Is there a linked Issue?

Manifests
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the directory's name containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/173463)

### CODE REVIEW

**Comment by stephengillie** [comment]
Automatic Validation ended with:

> Installation failed with exit code -1978334972
> 2024-09-15 07:16:06.646 [CLI ] No suitable installer found for manifest Microsoft.VCRedist.2015+.x64 with version 14.40.33810.0

Automated error analysis suggests -1978334972 may mean This package has a dependency missing from your system.

(Automated response - build 895.)

File: manifests/c/Celestia/Celestia/1.6.4/Celestia.Celestia.locale.en-US.yaml [hunk 1]
```diff
@@ -1,35 +1,34 @@
...
+  The free space simulation that lets you explore our universe in three dimensions.
+Moniker: celestia
+Tags:
+- celestia
+- foss
+- open-source
+- astronomy
+- educational
+- planetarium
+ReleaseNotesUrl: https://github.com/CelestiaProject/Celestia/releases/tag/1.6.4
```
Comment (stephengillie) [inline manifests/c/Celestia/Celestia/1.6.4/Celestia.Celestia.locale.en-US.yaml:29]:
```suggestion
ReleaseNotes: |-
  - Update site URL to celestiaproject.space
  - Fix building with Lua 5.4
ReleaseNotesUrl: https://github.com/CelestiaProject/Celestia/releases/tag/1.6.4
```

**Review by ItzLevvie — APPROVED** [review]

**Comment by stephengillie** [comment]
Automatic Validation ended with:

> Installation failed with exit code -1978334972
> 2024-09-17 00:53:49.884 [CLI ] No suitable installer found for manifest Microsoft.VCRedist.2015+.x64 with version 14.40.33810.0

Automated error analysis suggests -1978334972 may mean This package has a dependency missing from your system.

(Automated response - build 895.)

**Pull request closed without merging**